styleguide/derek/_partials/cards/corner-pocket.ejs

```diff
@@ -0,0 +1,7 @@
+<div class="cornerPocket-wrapper">
+  <div class="cornerPocket-borderWrapper">
+  	<h3 class="cornerPocket-heading">Starting at</h3>
+  	<h2 class="cornerPocket-callout">$0.015/hr</h2>
```

Comment:
Do these need to be `<h2>`/`<h3>` tags? I'm not sure this makes semantic sense, as these aren't really headings.